### PR TITLE
21131 - Basic STRR tile add to auth-web for account creation

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/21131

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/migrations/README [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Generic single-database configuration.
 
-1. Proper usage of migrations: python manage.py db migrate
+1. Add a new migration: /migrations  poetry run alembic revision --autogenerate -m "migration message"
+
+2. Proper usage of migrations: python manage.py db migrate
```
Comment (seeker25) [inline auth-api/migrations/README:5]: I don't think we have manage.py anymore? gun it?

File: auth-api/src/auth_api/resources/v1/products.py [hunk 2]
```diff
@@ -22,12 +22,14 @@
...
 from auth_api.services import Product as ProductService
 from auth_api.utils.endpoints_enums import EndpointEnum
+from auth_api.utils.auth import jwt as _jwt
 
 bp = Blueprint("PRODUCTS", __name__, url_prefix=f"{EndpointEnum.API_V1.value}/products")
 
 
 @bp.route("", methods=["GET", "OPTIONS"])
 @cross_origin(origins="*", methods=["GET"])
+@_jwt.requires_auth
```
Comment (seeker25) [inline auth-api/src/auth_api/resources/v1/products.py:32]: was this missing before?

Comment (seeker25) [reply auth-api/src/auth_api/resources/v1/products.py:32]: I see this might fail?

image

Comment (Jxio) [reply auth-api/src/auth_api/resources/v1/products.py:32]: yeah it was missing

**Review by seeker25 — APPROVED** [review]

**Pull request merged**